Packs/PaloAltoNetworks_IoT3rdParty/Scripts/GetCiscoISEActiveInstance/README.md

```diff
@@ -0,0 +1,38 @@
+This scripts determines which configured Cisco ISE instance is in active/primary state and returns the name of this instance.
```

Comment:
Determines which configured Cisco ISE instance is in active/primary state and returns the name of the instance.